lib/zendesk_apps_support/location.rb

```diff
@@ -3,8 +3,12 @@ class Location
     extend ZendeskAppsSupport::Finders
     attr_reader :id, :name, :orderable, :product_code
 
+    IDS = Set.new
```

Comment:
Right now it may be, but what prevents a new location from being created from anywhere else, causing another call to `add`?